packages/browser/src/plugins/segmentio/normalize.ts

```diff
@@ -9,6 +9,42 @@ import { SegmentioSettings } from './index'
 import { version } from '../../generated/version'
 import { getAvailableStorageOptions, UniversalStorage } from '../../core/user'
 
+declare global {
+  interface Navigator {
+    userAgentData: UserAgentData
```

Comment:
Maybe we just vendor these types (not npm install, just copy)? https://www.npmjs.com/package/user-agent-data-types/v/0.3.1?activeTab=code